pkg/cmd/delete.go

```diff
@@ -139,14 +139,14 @@ func getIntegration(ctx context.Context, c client.Client, name string, namespace
 	return &answer, nil
 }
 
-func deleteIntegration(ctx context.Context, c client.Client, integration *v1.Integration) error {
+func deleteIntegration(cmd *cobra.Command, ctx context.Context, c client.Client, integration *v1.Integration) error {
 	deleted, binding, err := deleteKameletBindingIfExists(ctx, c, integration)
 	if err != nil {
 		return err
 	}
 	if deleted {
 		// Deleting KameletBinding will automatically clean up the integration
-		fmt.Println("KameletBinding " + binding + " deleted")
+		fmt.Fprintln(cmd.OutOrStdout(),"KameletBinding " + binding + " deleted")
```

Comment:
Missing space before the "KameletBinding" parameter.

Reply:
fixed